src/style.css

```
@@ -0,0 +1,144 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  padding: 16px;
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 40px;
+}
+
+.card__title {
+  width: 166px;
+  height: 36px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+
+}
+
+.card__code {
+  width: 92px;
+  height: 14px;
+  line-height: 14px;
+  font-weight: 400;
+  font-size: 10px;
+  top: 247px;
+  left: 16px;
+  margin-bottom: 16px;
+  color: #616070;
+
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

why do you need this?
```suggestion
.stars__star:last-child {
  background-image: url(images/star.svg);
  background-position: center;
  background-repeat: no-repeat;
}
```